pkg/agent/core/ngt/service/vqueue/queue.go

```diff
@@ -180,7 +190,13 @@ func (v *vqueue) PushDelete(uuid string, date int64) error {
 	if date == 0 {
 		date = time.Now().UnixNano()
 	}
-	v.udim.Store(uuid, date)
+	odate, loaded := v.udim.LoadOrStore(uuid, date)
```

Comment:
maybe delete should be same as insert: 
if there is existing element with inserted later(or at the same time) but processed before the delete element, the delete operation should be void.
```suggestion
         idate, ok := v.uiim.Load(uuid)
	if ok && idate >= date {
		return nil
	}
	odate, loaded := v.udim.LoadOrStore(uuid, date)
```

Reply:
The Delete operation is prioritized at all times because it causes problems when executing updates.